python/python/glide/async_commands/core.py

```diff
@@ -1406,6 +1401,77 @@ async def zpopmin(
             ),
         )
 
+    async def zrange(
+        self,
+        key: str,
+        range_query: Union[RangeByIndex, RangeByLex, RangeByScore],
+        reverse: bool = False,
+    ) -> List[str]:
+        """
+        Returns the specified range of elements in the sorted set stored at `key`.
+
+        ZRANGE can perform different types of range queries: by index (rank), by the score, or by lexicographical order.
+
+        See https://redis.io/commands/zrange/ for more details.
```

Comment:
Please also add a note saying that to get the elements with their scores see zrange_withscores command